components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -333,6 +343,11 @@ const deleteFacilityOrHealthcareProfessional = async () => {
     }
 }
 
+const exitWithoutSavingUpdates = () => {
+    router.push('/moderation')
+    moderationScreenStore.setActiveScreen(ModerationScreen.Dashboard)
+}
+
```

Comment:
It looks like you need to rebase off main since you created a similar function in PR #1148